demo/python/leopard_demo_file.py

```diff
@@ -17,16 +17,31 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--access_key', required=True)
-    parser.add_argument('--library_path', default=None)
     parser.add_argument('--model_path', default=None)
+    parser.add_argument('--library_path', default=None)
+    parser.add_argument('--enable_automatic_punctuation', action='store_true')
+    parser.add_argument('--verbose', action='store_true')
     parser.add_argument('--audio_paths', nargs='+', required=True)
     args = parser.parse_args()
 
-    o = create(access_key=args.access_key, library_path=args.library_path, model_path=args.model_path)
+    o = create(
+        access_key=args.access_key,
+        model_path=args.model_path,
+        library_path=args.library_path,
+        enable_automatic_punctuation=args.enable_automatic_punctuation)
 
     try:
         for audio_path in args.audio_paths:
-            print(o.process_file(audio_path))
+            transcript, words = o.process_file(audio_path)
+            print(transcript)
+            if args.verbose:
+                for word in words:
+                    print('{')
+                    print('  word: "%s",' % word.word)
+                    print('  start_sec: %.2f,' % word.start_sec)
+                    print('  end_sec: %.2f,' % word.end_sec)
+                    print('  confidence: %.2f,' % word.confidence)
+                    print('}')
```

Comment:
In the other demos we're printing in more of a table format (each word object having it's own line) which makes it less of a vertical print